class P

	def main
		i = 1
		branch i
			on 1, pass
			on 2, pass
			on 1, pass  # .error. Branch case "1" was already used.
